packages/core/src/client.ts

```diff
@@ -3875,22 +3885,30 @@ export class MedplumClient extends TypedEventTarget<MedplumClientEventMap> {
   /**
    * Makes a POST request to the tokens endpoint.
    * See: https://openid.net/specs/openid-connect-core-1_0.html#TokenEndpoint
-   * @param formBody - Token parameters in URL encoded format.
+   * @param params - Token parameters.
    * @returns The user profile resource.
    */
-  private async fetchTokens(formBody: URLSearchParams): Promise<ProfileResource> {
+  private async fetchTokens(params: Record<string, string>): Promise<ProfileResource> {
+    const formBody = new URLSearchParams(params);
+    const headers: HeadersInit = { 'Content-Type': ContentType.FORM_URL_ENCODED, ...this.defaultHeaders };
```

Comment:
```suggestion
    const headers: HeadersInit = { ...this.defaultHeaders, 'Content-Type': ContentType.FORM_URL_ENCODED };
```
We probably don't want default headers to clobber anything we are setting in-line, though this is probably pedantic

Reply:
You're right, it makes more sense this way :+1: